forwards_paper.tex

```diff
@@ -125,21 +141,22 @@ \section*{Introduction}
 This embellished graph is known as the \emph{ancestral recombination graph}, or {ARG} \citep{griffiths}.
 Combined with ancestral genotypes and the origins of new mutations,
 it completely specifies the genomic sequence of any individual in the population at any time.
-However, much less than the entire ARG is needed to specify relationships 
+However, much less than the entire ARG is needed to specify relationships
 between any given set of samples --
-only those portions of it from which those samples have actually inherited, 
+only those portions of it from which those samples have actually inherited,
 back to their most recent common ancestors.
-The assumptions of coalescent theory 
+The assumptions of coalescent theory
 % -- random mating and neutrality --
 imply that this random set describes a Markov process looked at \emph{backwards} in time,
-and can hence be simulated without reference to the unnecessary remainder 
+and can hence be simulated without reference to the unnecessary remainder
 \citet{griffithsmarjoram,msprime}.
 This stochastic process is also called the ``ARG'',
 but we reserve this name for the embellished graph itself.
 % Although this point was in principle known since \citet{griffithsmarjoram},
 % only recent algorithmic advances made it possible to actually simulate the process correctly,
 % without approximation, across whole chromosomes \citep{msprime}.
-
+\jk{I find this confusing I'm afraid. I agree we should mention the ARG but I think only in passing.
+I have never found a situation in which an ARG made things easier to understand!}
```

Comment:
... but "ARG" is the name for what we are recording?  Can you elaborate what you mean here?

Reply:
I think the aside that "ARG actually means another thing too" (lines 153-154) could be confusing and isn't needed. I find the term ARG itself seems like helpful shorthand for "the entire history of ancestry and recombination" and think we should use it consistently for that within the paper

Reply:
To explain my perspective a bit, I personally find the term ARG to be unhelpful jargon at this point. It means at least four different things: a big ARG realisation of everything that happened to a sample; a little ARG realisation of the genetic ancestry of a sample; the big ARG stochastic process; and the little ARG stochastic process (== coalescent with recombination). So, my tendency is to avoid using it, except in order to link up with existing literature. 

That's just my anti-ARG prejudice though, and I don't want to impose this view on the paper. My specific comment here is that I find the description a bit confusing. I would have found it clearer if we said we record the population pedigree forwards in time, and then reduce this down to a minimal representation of genetic ancestry (mentioning in passing the connections with ARGs).

Again though, this is just my perspective. I'm not arguing we should definitely do it this way for the paper. I'm happy to go with the consensus view on this issue!

Reply:
Well, we need to link up to that literature somehow.  My position is that we record more than the pedigree, and so we need a name for that.  I suppose that "tree sequence" carries equivalent information, but has a left-to-right connotation rather than the forwards/backwards in time connotation.

I'm collecting data on the issue: https://twitter.com/petrelharp/status/921782447230017536